### DISCUSSION TITLE
Pending channel close since several days

### DISCUSSION THREAD

**Discussion by GreenPepe21** [Ride-The-Lightning/RTL#807] [Troubleshooting Questions] [answered]
I attempted to close a channel I had with a node (not force-close), which is stuck as "waiting close" since several days now. The other node is online (node ID: 02ab9dcdfb878badeee6a1872aea72acd6ed55c276a63419a3788a4551bf2934c8) but I do not know why the channel does not close...
Anyone had the same issue or knows a way to resolve this? Can I force-close a channel that is waiting close?
Also what is the difference between "pending close" and "waiting close"? 
Sorry for the noob questions and thanks for any help

**Comment by saubyk** [comment, marked as answer]
Hi @GreenPepe21 

Typically 'Waiting Close' status is for the situation, where the node is waiting for the closing transaction to be mined (to be included on a block). Once the transaction has been mined and you need to wait for to receive your funds due to time lock e.g. in case the close type was force close, the settlement of the funds waits in 'Pending Close' status.

In case your node is not resolving the status from Waiting Close to Pending Close, you should confirm the below things:
- try restarting your node
- verify if there was a local balance that you're expecting on the channel closure and if that amount is now available in your on-chain balance.
- If the above is true, it might just be a LND issue where its not resolving the channel status to the correct one, but your funds are safely back in your custody

If none of the above is clear from your investigation, you should contact the LND team on their slack channel or open an issue on their GitHub.

Hope this helps.

**Reply by GreenPepe21** [reply]
Thanks. Unfortunately these methods didn't help. Will write the LND team...
